tests/test_models/test_sarimax_model.py

```diff
@@ -343,3 +357,11 @@ def test_params_to_tune(model, example_tsds):
     ts = example_tsds
     assert len(model.params_to_tune()) > 0
     assert_sampling_is_valid(model=model, ts=ts)
+
+
+def test_fit_params_passed_to_fit_method(example_tsds):
+    model = SARIMAXModel(fit_params={"disp": False})
+    fit = spy_decorator(SARIMAX.fit)
```

Comment:
In fact you can do sth like this
```
 with patch("statsmodels.tsa.statespace.sarimax.SARIMAX.fit", Mock()):
        model.fit(example_tsds)
        SARIMAX.fit.assert_called_with(disp=False)
```
I sent you a little bit over complicated example